src/components/pages/SearchResultsPage.js

```diff
@@ -59,13 +60,13 @@ const SearchResultsPage = () => {
               <p className="inline text-light-gray font-light"> Today</p>
             </p>
           </div>
-          {/* <div className="flex"></div> */}
           <div className="w-11/12 h-4/5 mx-auto my-10">
             <Graph
               chartData={chartData}
               categories={categories}
               getData={getData}
             />
+            <StockTable />
```

Comment:
We should pass the stock data to the StockTable component as a prop, similar to how charData is passed into the Graph component.